Backpack-SwiftUI/Skeleton/Classes/BPKShimmer.swift

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct BPKShimmer: ViewModifier {
+    @State private var phase: CGFloat = 0
+    var duration = 1.0
+    var bounce = false
+    var delay = 0.2
```

Comment:
Are these adjustable by the end user? I feel like we rather make these private let?

Reply:
I think yes, I exposed it, see: https://github.com/Skyscanner/backpack-ios/pull/1393/files#diff-38c5c9da4eeebd536cbba61381c3f23868e1fd2f9deb5a663fb7f690a4c590afR70

User can modify this by `.shimmering(animated: , duration: , delay:, bounce: ) `

Reply:
Should we allow this customization? I feel like we want to control how the shimmering works
@frugoman ?

Reply:
Yea, I can also make it as unchangeable, it's up to you.

Reply:
I think this should not be adjustable by the end user, only by Backpack, so that's consistent all across the product. Lets make this unchangeable